MHCollapsibleViewManager.h

```diff
@@ -0,0 +1,65 @@
+//
+//  MHCollapsibleViewManager.h
+//  Represents a view piece that is collapsible
+//  It manages the following:
+//      The header record style/accessory for clicked record
+//      Showing/Hiding rows under header
+//      Clicked child cell style/accessory
+//      Keeps track if cell was clicked using MHFilterLabel class
+//
+//  Created by Lizzy Randall on 5/6/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@interface MHCollapsibleViewManager : NSObject
+
+@property (strong, nonatomic) NSMutableArray *data;
+//Stores four uitableview cells with properties to change style, accessory, etc.
+@property (strong, nonatomic) NSArray *cellStyles;
+//Shown on header and also can be used to determine logic to name the manager
+//i.e. headerTitle of Labels means label manager
+@property (strong, nonatomic) NSString *headerTitle;
+
+@property (nonatomic) UITableViewRowAnimation rowAnimation;
```

Comment:
I would recommend moving a number of these definitions to the .m file. objc doesn't have public or private keywords for making properties and methods public or private instead they ask you to put public methods and properties in the .h file and private properties and methods in an anonymous interface in the .m file.
eg
MHCollapsibleViewManager.h
@interface MHCollapsibleViewManager : NSObject

@property (strong, nonatomic) NSMutableArray *data;
@property (nonatomic) NSInteger numOfRows;
+@property (nonatomic) BOOL expanded;

- (id)initalizeManager:(UITableViewRowAnimation)animation title:(NSString*) title tableView: (UITableView*)tableView;
+- (void) initializeData:(NSMutableArray*)dataFromController;
- (void)setCellProperties:(UITableViewCell*)clickedHeader header:(UITableViewCell*)header clickedCell:(UITableViewCell*)clickedCell normalCell:(UITableViewCell*)cell tableView:(UITableView*)tableView;
- (UITableViewCell*)setCellSettings:(UITableViewCell*)cell type:(NSString*)type;
- (UITableViewCell*)returnCell:(NSIndexPath*)indexPath tableView:(UITableView*)tableView;
- (NSMutableArray*)returnSelectedRows;
- (void) toggleCollapse: (UITableView*)tableView indexPath:(NSIndexPath*)indexPath;
- (void) selectedRowAtIndexPath: (UITableView*)tableView indexPath:(NSIndexPath*)indexPath

@end

MHCollapsibleViewManager.m

above your implementation portion of the file you would put
@interface MHCollapsibleViewManager ()

@property (strong, nonatomic) NSArray *cellStyles;
@property (strong, nonatomic) NSString *headerTitle;
@property (nonatomic) UITableViewRowAnimation rowAnimation;

//any private methods you might want to define

@end